pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -272,6 +281,14 @@ func (h *upgradeHandler) OnChanged(key string, upgrade *harvesterv1.Upgrade) (*h
 				return h.upgradeClient.Update(toUpdate)
 			}
 		} else {
+			// save the original value of replica-replenishment-wait-interval setting and extend it with a longer value
```

Comment:
@starbops Please help add a check: If the value is already more than 30 minutes (e.g., the user set it), let's not change it.

Reply:
Updated, thanks.